app/src/utils.ts

```diff
@@ -22,17 +22,37 @@ export function slåSammenTilFulltNavn({
   mellomnavn?: string;
   etternavn?: string;
 }) {
-  return [fornavn, mellomnavn, etternavn].filter(Boolean).join(" ");
+  return formatNavn([fornavn, mellomnavn, etternavn].filter(Boolean).join(" "));
 }
 
-export function navnMedStorBokstav(navn?: string) {
+export function formatNavn(navn?: string) {
   if (!navn) {
     return navn;
   }
   return navn
-    .split("-")
-    .map((old) => capitalize(old))
-    .join("-");
+    .toLowerCase()
+    .split(" ")
+    .map((del) =>
+      del
+        .split("-")
+        .map((delnavn) => capitalize(delnavn))
+        .join("-"),
+    )
+    .join(" ");
+}
+
+export function formatTelefonnummer(telefonnummer: string): string {
+  const tegn = telefonnummer.replaceAll(/\D/g, "");
+
+  if (tegn.length !== 8) {
+    return telefonnummer;
+  }
+
+  if (["4", "8", "9"].includes(tegn[0])) {
+    return `${tegn.slice(0, 3)} ${tegn.slice(3, 5)} ${tegn.slice(5)}`;
+  }
+
+  return `${tegn.slice(0, 2)} ${tegn.slice(2, 4)} ${tegn.slice(4, 6)} ${tegn.slice(6)}`;
 }
```

Comment:
Ref denne: https://www.korrekturavdelingen.no/tall-siffer-gruppering.htm

Det virker som om folk krangler litt om mobilnumre skal skrives 2 2 2 2 eller 3 2 3. Åpen for å diskutere denne – gikk i alle fall for 3 2 3.